develop/tutorials/articles/260-themes-and-layout-templates/xx-legacy-ant-theme-tasks.markdown

```diff
@@ -0,0 +1,188 @@
+Here's the format to follow in executing the script: 
+
+    create.[sh|bat] [project-name] "[Theme Display Name]"
+
+On Linux and Mac OS X, you'd enter a command similar to the one in this example:
+
+    ./create.sh my-theme "My Theme"
+        
++$$$
+
+**Note:** You may not be able to run the *create* script right away. If that is 
+the case, you'll need to make the `create.sh` script executable. To make the
+script executable, run the following command in the terminal, from within the 
+`themes` folder of the Plugins SDK: `chmod +x create.sh`. Now you should be 
+able to run the script without any problems.
+
+$$$
+
+On Windows, you'd enter a command similar to the one in this example:
+
+    create.bat my-theme "My Theme"
+    
+Your theme project is a blank slate on which you you can exercise your creative 
+talents. The Plugins SDK automatically appends "-theme" to your project name. 
+Right now your theme is empty. Custom themes are created by layering your 
+customizations on top of one of Liferay's built-in themes.
+
+You can take a closer look at the theme anatomy next.
+
+### Plugins SDK Theme Anatomy [](id=plugins-sdk-theme-anatomy)
+
+Once you've created your theme, you should end up with the structure below:
+
+- `[theme-project-name]/`
+    - `docroot/`
+        - `_diffs/` (subfolders not created by default)
+            - `css/`
+            - `images/`
+            - `js/`
+            - `templates/`
+        - `css/`
+                - `_application.scss`
+                - `_aui_custom.scss`
+                - `_aui_variables.scss`
+                - `_base.scss`
+                - `_control_menu.scss`
+                - `_custom_common.scss`
+                - `_custom.scss`
+                - `_extras.scss`
+                - `_imports.scss`
+                - `_layout.scss`
+                - `_liferay_custom.scss`
+                - `_liferay_variables_custom.scss`
+                - `_liferay_variables.scss`
+                - `_navigation.scss`
+                - `_portal.scss`
+                - `_portlet.scss`
+                - `_taglib.scss`
+                - `application`
+        	- `aui/`
+        		- (many directories)
+        	- `aui.scss`
+        	- `base/`
+        	- `layout/`
+        	- `main.scss`
+        	- `navigation/`
+        	- `portal/`
+        	- `portlet/`
+        	- `taglib/`
+        - `images/`
+            -   (many directories)
+        - `js/`
+            - `main.js`
+        - `templates/`
+            - `init_custom.ftl`
+            - `navigation.ftl`
+            - `portal_normal.ftl`
+            - `portal_pop_up.ftl`
+            - `portlet.ftl`
+        - `WEB-INF/`
+        	- `liferay-look-and-feel.xml`
+            - `liferay-plugin-package.properties`
+
+If you're familiar with theme development in 6.2, you may have noticed a few
+changes in the directory structure and filenames.
+
+Starting in Liferay Portal 7.0 and Liferay DXP, all theme CSS files use SASS and
```

Comment:
Ok. I'll write something along the lines of "Starting in Liferay Portal 7.0 and Liferay DXP, all theme CSS files have to follow the SASS naming convention, using the `.scss` file extension..."